src/datasets/streaming.py

```diff
@@ -98,6 +99,9 @@ def wrapper(*args, **kwargs):
     patch_submodule(module, "pd.read_csv", wrap_auth(xpandas_read_csv), attrs=["__version__"]).start()
     patch_submodule(module, "pd.read_excel", xpandas_read_excel, attrs=["__version__"]).start()
     patch_submodule(module, "sio.loadmat", wrap_auth(xsio_loadmat), attrs=["__version__"]).start()
+    # xml.dom.minidom
+    if hasattr(module, "parse") and module.parse.__module__ == "xml.dom.minidom":
+        patch_submodule(module, "parse", wrap_auth(xxml_dom_minidom_parse)).start()
```

Comment:
Btw with the changes in https://github.com/huggingface/datasets/pull/4375 we'll soon be able to do
```python
patch_submodule(module, "xml.dom.minidom.parse", wrap_auth(xxml_dom_minidom_parse)).start()
```
without an `if` and it will work even if it's imported like `from xml.dom.minidom import parse` :)

Reply:
Much better!!! :heart:  Current way it too cumbersome...